lib/components/src/blocks/ArgsTable/ArgControl.tsx

```diff
@@ -21,10 +23,12 @@ export interface ArgControlProps {
 export const ArgControl: FC<ArgControlProps> = ({ row, arg, updateArgs }) => {
   const { name, control } = row;
   const onChange = useCallback(
-    (argName: string, argVal: any) => {
-      updateArgs({ [name]: argVal });
-      return argVal;
-    },
+    debounce(
```

Comment:
@ndelangen in knobs there is a 325ms debounce. but there's also an option to force update without debounce. do we need that for args?

Reply:
the knobs debounce is a patch to bad design, not something we ever wanted.

If we can make args work without it, it will lead to much more predictable behavior I think.

Reply:
Hmm. Technically speaking Args "works" without it, but it's not performant, and I can see why the debounce is desired. I'm not sure what kind of edge cases this brings up.

Reply:
If the props table is too long, it is very heavy to edit every field, especially when users write some urls in configs, and use those urls to fetch from back-end, every edit can cause a fetch